modules/core/src/core-layers/text-layer/font-atlas.js

```diff
@@ -36,26 +36,31 @@ export function makeFontAtlas(
   // measure texts
   let row = 0;
   let x = 0;
+  let fontHeight = 0;
   const mapping = {};
 
   Array.from(characterSet).forEach(char => {
-    const {width} = ctx.measureText(char);
+    const {width, fontBoundingBoxDescent} = ctx.measureText(char);
+    if (!fontHeight) {
+      fontHeight = fontBoundingBoxDescent;
+    }
+
     if (x + width > MAX_CANVAS_WIDTH) {
       x = 0;
       row++;
     }
     mapping[char] = {
       x,
-      y: row * (fontSize + padding),
+      y: row * (fontHeight + padding),
       width,
-      height: fontSize,
+      height: fontHeight,
       mask: true
     };
     x += width + padding;
   });
 
   canvas.width = MAX_CANVAS_WIDTH;
-  canvas.height = (row + 1) * (fontSize + padding);
+  canvas.height = (row + 1) * (fontHeight + padding);
```

Comment:
I guess fontBoundingBoxDescent will be different for each char?

if that's the case, in line 63 we are using the fontHeight of the last chart to define the canvas height?
shall we use Math.max(...fontBoundingBoxDescents)?

else if fontBoundingBoxDescent would be the same for all chars, we don't need to measure it for each chars? (nvm if it is a cheap operation)